bokehjs/src/coffee/models/glyphs/vbar.coffee

```diff
@@ -0,0 +1,67 @@
+_ = require "underscore"
+rbush = require "rbush"
+Quad = require "./quad"
+Glyph = require "./glyph"
+hittest = require "../../common/hittest"
+p = require "../../core/properties"
+
+class VBarView extends Glyph.View
+
+  # Temporarily commented out. To be put back later.
+  # _index_data: () ->
+  #   index = rbush()
+  #   pts = []
+  #   for i in [0...@x.length]
+  #     if not isNaN(@x[i] + @width[i] + @top[i] + @bottom[i])
+  #       w2 = @width[i]/2
+  #       pts.push([@x[i]-w2, @bottom[i], @x[i]+w2, @top[i], {'i': i}])
+  #   index.load(pts)
+  #   return index
```

Comment:
did you want to continue this in this PR? If not, let's just delete this and the other block in the other file. I'm almost always -1 on checking in commented-out code